projects/standard-rulesets/src/examples/requireExample.ts

```diff
@@ -7,6 +7,7 @@ import {
 } from '@useoptic/rulesets-base';
 import { appliesWhen } from './constants';
 import { OpenAPIV3 } from 'openapi-types';
+import { qualifiedContentType } from './qualifiedContentType';
 
 export const requireResponseExamples = (applies: typeof appliesWhen[number]) =>
   new ResponseRule({
```

Comment:
Now that I read this again, wouldn't this be better as a ResponseBody type? Otherwise this would trigger on any response change, not just a body change?

And that way, you can scope this rule to specific content types